enterprise/backend/src/metabase_enterprise/advanced_config/util/urls.clj

```diff
@@ -0,0 +1,14 @@
+(ns metabase-enterprise.advanced-config.util.urls
+  (:require [metabase.models.setting :as setting :refer [defsetting]]
+            [metabase.public-settings :as public-settings]
+            [metabase.public-settings.premium-features :as premium-features]
+            [metabase.util.i18n :refer [deferred-tru]]))
+
+(defsetting pulse-url
+  (deferred-tru "By default \"Site Url\" is used in notification links, but can be overridden."))
+
+(defn site-url
+  "Return the Pulse URL if set, or Site URL."
+  []
+  (when (premium-features/enable-advanced-config?)
```

Comment:
@camsaul Should this be under the `advanced_config` token or the `embedding` token?

Reply:
Yeah I think it might make sense to have this feature flag on `embedding` since it's supposed to be something that only works for premium embedding. The util functions should go in a `metabase-enterprise.embedding.util` namespace or something probably